themes/bootstrap5/templates/RecordTab/holdingsils/standard.phtml

```diff
@@ -30,6 +30,9 @@
       </span>
       <?php if ($availabilityStatus->isAvailable()): ?>
         <?php /* Begin Available Items (Holds) */ ?>
+        <?php if (isset($holding['loan_type_id']) && !empty($holding['loan_type_name'])): ?>
```

Comment:
Is it necessary to check loan_type_id? If loan_type_name is the only thing you're displaying, it's probably fair to say it's the only thing that needs checking.

However, we should probably come up with a mechanism for enabling or disabling this display, since not everyone is going to want it. Any thoughts/preferences on whether it would make more sense to put something in Folio.ini or in a more general place like config.ini? (I see pros/cons to both -- if we put the setting in Folio.ini, then we can totally bypass the extra data lookup when it is not wanted and save some time/cycles... but if we put the setting in config.ini, then it's a more universal setting and will carry across all ILS drivers without having to implement things independently in all of them).